### PR TITLE
Add image mirroring for aws and azure cloud controller managers

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

**Comment by Fedosin** [comment]
It's mainly for cloud-provider-aws and cloud-provider-azure

File: core-services/image-mirroring/openshift/mapping_origin_4_9 [hunk 1]
```diff
@@ -248,18 +257,24 @@ registry.ci.openshift.org/origin/4.9:telemeter quay.io/openshift/origin-telemete
...
 registry.ci.openshift.org/origin/4.9:vertical-pod-autoscaler quay.io/openshift/origin-vertical-pod-autoscaler:4.9 quay.io/openshift/origin-vertical-pod-autoscaler:4.9.0
 registry.ci.openshift.org/origin/4.9:vertical-pod-autoscaler-operator quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.9 quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.9.0
+registry.ci.openshift.org/origin/4.9:vmware-vsphere-csi-driver quay.io/openshift/origin-vmware-vsphere-csi-driver:4.9 quay.io/openshift/origin-vmware-vsphere-csi-driver:4.9.0
+registry.ci.openshift.org/origin/4.9:vmware-vsphere-csi-driver-operator quay.io/openshift/origin-vmware-vsphere-csi-driver-operator:4.9 quay.io/openshift/origin-vmware-vsphere-csi-driver-operator:4.9.0
+registry.ci.openshift.org/origin/4.9:vmware-vsphere-csi-driver-operator-test quay.io/openshift/origin-vmware-vsphere-csi-driver-operator-test:4.9 quay.io/openshift/origin-vmware-vsphere-csi-driver-operator-test:4.9.0
+registry.ci.openshift.org/origin/4.9:vmware-vsphere-csi-driver-syncer quay.io/openshift/origin-vmware-vsphere-csi-driver-syncer:4.9 quay.io/openshift/origin-vmware-vsphere-csi-driver-syncer:4.9.0
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_9:271]: This image needs to be pruned (was renamed)

File: core-services/image-mirroring/openshift/mapping_origin_4_7 [hunk 2]
```diff
@@ -207,6 +210,7 @@ registry.ci.openshift.org/origin/4.7:openstack-cloud-controller-manager quay.io/
 registry.ci.openshift.org/origin/4.7:openstack-installer quay.io/openshift/origin-openstack-installer:4.7 quay.io/openshift/origin-openstack-installer:4.7.0
 registry.ci.openshift.org/origin/4.7:openstack-machine-controllers quay.io/openshift/origin-openstack-machine-controllers:4.7 quay.io/openshift/origin-openstack-machine-controllers:4.7.0
 registry.ci.openshift.org/origin/4.7:operator-lifecycle-manager quay.io/openshift/origin-operator-lifecycle-manager:4.7 quay.io/openshift/origin-operator-lifecycle-manager:4.7.0
+registry.ci.openshift.org/origin/4.7:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.7 quay.io/openshift/origin-operator-marketplace:4.7.0
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_7:213]: I’m not sure this is correct

**Comment by Fedosin** [comment]
@smarterclayton I left only required images and removed everything else. It should be better now.

**Pull request merged**